cmd/kubelet/app/server.go

```diff
@@ -207,6 +208,10 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 				klog.Fatal(err)
 			}
 
+			if 0 != strings.Index(kubeletConfig.KubeletCgroups, kubeletConfig.KubeReservedCgroup) {
```

Comment:
I think we want to output this error only when both kubeletcgroups and kubereservedcgroup are both set.

Reply:
right，i also think that's better. i added the judgement.